ngraph/src/ngraph/op/interpolate.hpp

```diff
@@ -136,18 +144,38 @@ namespace ngraph
                     NearestMode nearest_mode = NearestMode::round_prefer_floor;
                     // a flag that specifies whether to perform anti-aliasing. default is `false`
                     bool antialias = false;
```

Comment:
You can remove this default values if you set their in the constructor.

Reply:
Ok, I will do it in next pr

Reply:
FIxed: https://github.com/openvinotoolkit/openvino/pull/1484/